src/api/hooks/useGetCoinSupplyLimit.tsx

```diff
@@ -29,9 +32,25 @@ interface AggregatorData {
 }
 
 async function fetchTotalSupply(
-  aggregatorData: AggregatorData,
+  data: Types.MoveResource[],
   state: GlobalState,
 ): Promise<number | null> {
+  const coinInfo = data.find(
```

Comment:
why not use `useGetAccountResource`? https://github.com/aptos-labs/explorer/blob/main/src/api/hooks/useGetAccountResource.ts

Reply:
good point